ezpublish/config/ezpublish_behat.yml

```diff
@@ -1,46 +1,2 @@
-doctrine:
-    dbal:
-        connections:
-            default:
-                driver: pdo_mysql
-                host: localhost
-                user: root
-                dbname: behattestdb
-                charset: UTF8
-
-ezpublish:
-    repositories:
-        behat: ~
-    siteaccess:
-        # Available siteaccesses
-        list:
-            - behat_site
```

Comment:
`ezpublish_test.yml` is for unit test use in src folder, ideally should be able to run w/o full setup if I remember how they use it in symfony standard correct. But lets deal with that later.